schema/2.0.0/corpora_schema.md

```diff
@@ -0,0 +1,264 @@
+
+[Mondo Disease Ontology]: http://obofoundry.org/ontology/mondo.html
+[2021-06-01]: https://github.com/monarch-initiative/mondo/releases/tag/v2021-06-01
+[mondo.owl]: https://github.com/monarch-initiative/mondo/releases/download/v2021-06-01/mondo.owl
+
+[Mouse Developmental Stages]: http://obofoundry.org/ontology/mmusdv.html
+[mmusdv.owl]: https://github.com/obophenotype/developmental-stage-ontologies/blob/0.1/src/mmusdv/mmusdv.owl
+
+[NCBI organismal classification]: http://obofoundry.org/ontology/ncbitaxon.html
+[2021-02-15]: https://github.com/obophenotype/ncbitaxon/releases/tag/v2021-02-15
+[ncbitaxon.owl]: https://github.com/obophenotype/ncbitaxon/releases/download/v2021-02-15/ncbitaxon.owl.gz
+
+[Phenotype And Trait Ontology]: http://www.obofoundry.org/ontology/pato.html
+[2021-05-26]: https://github.com/pato-ontology/pato/releases/tag/v2021-05-26
+[pato.owl]: https://github.com/pato-ontology/pato/blob/v2021-05-26/pato.owl
+
+[Uberon multi-species anatomy ontology]: http://www.obofoundry.org/ontology/uberon.html
+[2021-02-12]: https://github.com/obophenotype/uberon/releases/tag/v2021-02-12
+[uberon.owl]: https://github.com/obophenotype/uberon/blob/v2021-02-12/uberon.owl
+
+### Required Gene Annotations
+
+The following gene annotation dependencies are *pinned* for this version of the schema.
+
+| Organism | Required version | Download |
+|:--|:--|:--|
+| [ENSEMBL (Human)] | Human reference GRCh38 (GENCODE v32/Ensembl 98)] matching [cellranger 2020-A (July 7, 2020) release] | [gencode.v32.primary_assembly.annotation.gtf] |
+| [ENSEMBL (Mouse)] | Mouse reference mm10 (GENCODE vM23/Ensembl 98) matching [cellranger 2020-A (July 7, 2020) release]| [gencode.vM23.primary_assembly.annotation.gtf] |
+|||
+
+[ENSEMBL (Human)]: http://www.ensembl.org/Homo_sapiens/Info/Index
+[gencode.v32.primary_assembly.annotation.gtf]:http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_32/gencode.v32.primary_assembly.annotation.gtf.gz"
+
+[ENSEMBL (Mouse)]: http://www.ensembl.org/Mus_musculus/Info/Index
+[gencode.vM23.primary_assembly.annotation.gtf]:http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M23/gencode.vM23.primary_assembly.annotation.gtf.gz"
+
+[cellranger 2020-A (July 7, 2020) release]: https://support.10xgenomics.com/single-cell-gene-expression/software/release-notes/build
+
+## `obs` (Cell Metadata)
+
+`obs` is a [`pandas.DataFrame`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html).
+
+Curators MUST annotate the following columns in the `obs` dataframe:
+
+| Key | Value | Annotator |
+|-|-|-|
+| assay_ontology_term_id | `str` or categorical with `str` categories. This MUST be a child of [EFO:0002694](https://www.ebi.ac.uk/ols/ontologies/efo/terms?short_form=EFO_0002694). If there is not an exact match for the assay, *clarifying text* MAY be appended in parentheses such as " (sci-plex)". | Curator |
```

Comment:
> Question: In instances where we require a child of a specific term, what do you think about also including the label of the ontology? I think it makes it a bit more readable for people who are browsing the ontology to know that we're requiring, e.g. a MONDO disease term or PATO:0000461 (Normal).

I rewrote similar examples using the pattern:

[CL:0000003](https://www.ebi.ac.uk/ols/ontologies/cl/terms?obo_id=CL:0000003) for "native cell"

because I was concerned that curators might be confused by the text and attempt to annotate `PATO:0000461 (Normal)`.

Reply:
How about:

This MUST be a child of [EFO:0002694](https://www.ebi.ac.uk/ols/ontologies/efo/terms?short_form=EFO_0002694) which is "experimental process".

or

This MUST be a child of "experimental process". See [EFO:0002694](https://www.ebi.ac.uk/ols/ontologies/efo/terms?short_form=EFO_0002694).

Reply:
Flipped a coin. Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/67/commits/a0909191d6f636f2c857ece47eae5537d1ef9223

Reply:
Your choice looks good to me.